worker/rhpv3.go

```diff
@@ -323,7 +323,10 @@ func (h *host) SyncAccount(ctx context.Context, revision *types.FileContractRevi
 	return h.acc.WithSync(ctx, func() (types.Currency, error) {
 		var balance types.Currency
 		err := h.transportPool.withTransportV3(ctx, h.HostKey(), h.siamuxAddr, func(t *transportV3) error {
-			payment := preparePayment(h.accountKey, pt.AccountBalanceCost, pt.HostBlockHeight)
+			payment, err := payByContract(revision, pt.AccountBalanceCost, h.acc.id, h.renterKey)
```

Comment:
This introduces an error which we probably want to fix before merging? For some reason the host processes a refund here, and if the ephemeral account is completely filled, it will return a balance exceeds error. If you pass an empty account id as refund account, you get another error indicating the empty account id is used for refund account.

Reply:
Should be fixed in `hostd` now.